routes/account/manage.ts

```diff
@@ -3,21 +3,14 @@ import type { RouteContext } from "$fresh/server.ts";
 import { stripe } from "@/utils/stripe.ts";
 import type { SignedInState } from "@/middleware/session.ts";
 import { redirect } from "@/utils/http.ts";
-import { createHttpError } from "std/http/http_errors.ts";
-import { Status } from "std/http/http_status.ts";
 
 export default async function AccountManagePage(
   _req: Request,
   ctx: RouteContext<undefined, SignedInState>,
 ) {
-  if (stripe === undefined) throw createHttpError(Status.NotFound);
-
   const { sessionUser } = ctx.state;
-  if (sessionUser.stripeCustomerId === undefined) {
-    throw createHttpError(
-      Status.NotFound,
-      "User does not have a Stripe customer ID",
-    );
+  if (stripe === undefined || sessionUser.stripeCustomerId === undefined) {
+    return ctx.renderNotFound();
```

Comment:
This is a page, ultimately redirecting, so error handling by the middleware is different than the error handling for APIs. Without rethinking the middleware I figured I would propose an easy general 404.

Reply:
Yeah, I was thinking about this too. I think there's no point in throwing `Status.NotFound` errors in the web page section of the site. I'll look into this separately, as I'll be creating an error-handling plugin.